rxbindings/src/main/java/com/amplifyframework/rx/RxApiBinding.java

```diff
@@ -26,16 +26,19 @@
 import com.amplifyframework.api.rest.RestOptions;
 import com.amplifyframework.api.rest.RestResponse;
 import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.core.async.Cancelable;
 import com.amplifyframework.rx.RxAdapters.CancelableBehaviors;
 
 import io.reactivex.rxjava3.core.Observable;
 import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
 
 /**
  * An implementation of the RxApiCategoryBehavior which satisfies the API contract by wrapping
  * {@link ApiCategoryBehavior} in Rx primitives.
  */
-final class RxApiBinding implements RxApiCategoryBehavior {
+public final class RxApiBinding implements RxApiCategoryBehavior {
```

Comment:
Is this access level escalation required, for the _implementation class_? (The public methods will still be public.)

Reply:
Yeah...that's because anything outside of the package wouldn't have access to the RxSubscriptionOperation object. That being said, rather than making this public, I created a top level class called RxOperations where we can put of the Rx-operation related objects. That, we can make public.